NuGet.Docs/Consume/NuGet-Config-File.md

```diff
@@ -288,3 +288,24 @@ This results in something more readable (or even manually configurable):
 <h3>API Key to access package source</h3>
 Allows you to set the API Key corresponding to a specific package source.
 This key  has to be set via <a href="/Consume/command-line-reference#Setapikey-Command">NuGet -SetApiKey</a>
+
+### Environment variables in configuration
+
+Starting with NuGet 3.4, NuGet evaluates environment variables in NuGet.config values.
```

Comment:
Looks good to me.  Thank you for your contribution!

We will merge this documentation when the source is merged.




On Sat, Jan 9, 2016 at 4:24 PM, Travis Smith <notifications@github.com>
wrote:

> In NuGet.Docs/Consume/NuGet-Config-File.md
> <https://github.com/NuGet/NuGetDocs/pull/431#discussion_r49268359>:
>
> > @@ -288,3 +288,24 @@ This results in something more readable (or even manually configurable):
> >  <h3>API Key to access package source</h3>
> >  Allows you to set the API Key corresponding to a specific package source.
> >  This key  has to be set via <a href="/Consume/command-line-reference#Setapikey-Command">NuGet -SetApiKey</a>
> > +
> > +### Environment variables in configuration
> > +
> > +Starting with NuGet 3.4, NuGet evaluates environment variables in NuGet.config values.
>
> I'm just guessing with 3.4 -- will that be right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NuGet/NuGetDocs/pull/431/files#r49268359>.
>